src/Compilers/CSharp/Test/Semantic/Semantics/RefExtensionMethodsTests.cs

```diff
@@ -982,7 +982,7 @@ public static void Main()
             var reference = CreateCompilationWithMscorlibAndSystemCore(code).VerifyDiagnostics(
                 // (4,24): error CS8338: The first parameter of the readonly reference extension method 'PrintValue' must be a value type.
                 //     public static void PrintValue<T>(in this T p) where T : System.IComparable
-                Diagnostic(ErrorCode.ERR_RefReadOnlyExtensionMustBeValueType, "PrintValue").WithArguments("PrintValue").WithLocation(4, 24),
+                Diagnostic(ErrorCode.ERR_InExtensionMustBeValueType, "PrintValue").WithArguments("PrintValue").WithLocation(4, 24),
```

Comment:
Could you add a test with a legacy Span (lacking `ref`)? It should no longer be recognized.